charts/ingress/values.yaml

```diff
@@ -44,3 +44,4 @@ gateway:
   env:
     role: traditional
     database: "off"
+    router_flavor: "expressions"
```

Comment:
Could we add a template to check image of KIC?

Reply:
Can you please elaborate on what you mean by this? There are two validation rules added in `validations.tpl` file: one for KIC < 2.10 and one for KIC < 3.0.